src/libraries/System.Memory/tests/Span/Replace.T.cs

```diff
@@ -36,57 +37,113 @@ protected ReplaceTests()
         }
 
         [Fact]
-        public void ZeroLengthSpan()
+        public void ZeroLengthSpan_InPlace()
         {
-            Exception actual = Record.Exception(() => Span<T>.Empty.Replace(_oldValue, _newValue));
+            Span<T>.Empty.Replace(_oldValue, _newValue);
```

Comment:
Presumably this is also changing behavior of the existing overload?

Reply:
There's no change in behavior here.  I just changed the test setup.  It was validating that no exception was being thrown by catching any exception and then using Assert.Null to check that none was actually caught.  I just removed all that such that if it throws an exception, that exception will propagate out of the test and fail it... that's not only a simpler test, it'd show us what the exception is if it fails :)